main.go

```diff
@@ -624,16 +638,6 @@ func (s *DB) NewRecord(value interface{}) bool {
 	return s.NewScope(value).PrimaryKeyZero()
 }
 
-// RecordNotFound check if returning ErrRecordNotFound error
-func (s *DB) RecordNotFound() bool {
```

Comment:
This is change #1

Reply:
Maybe, for easier backwards comp, we could supply a new implementation based on the new `errors.Is` approach. Something like 
```
func (s *DB) RecordNotFound() bool {
return errors.Is(s.Error, gorm.ErrRecordNotFound)
}
```

Reply:
Sure but I don't want backward compatibility at all, I want to get rid of the fork.

Reply:
oh, I thought this fork was adding some value to the original Gorm, but it in fact it does not. Nevermind me then, let's kill the fork.

Reply:
Initially, it did add some value (@michalg9 had a bugfix), but then the original author completely rewrote GORM from scratch. Michal's bugfix is not relevant anymore.

Reply:
It did add value. The bugfix was about propagating nested preload errors (so e.g. "Stack.Account") to the top (basically if there was an error in such preload, we would not receive it from the query). I can imagine in gormv2 this was fixed, and if we do not have to use the fork anymore, let's not :D